core/schema/directory.go

```diff
@@ -80,7 +80,8 @@ func (s *directorySchema) Install() {
 		dagql.Func("export", s.export).
 			Impure("Writes to the local host.").
 			Doc(`Writes the contents of the directory to a path on the host.`).
-			ArgDoc("path", `Location of the copied directory (e.g., "logs/").`),
+			ArgDoc("path", `Location of the copied directory (e.g., "logs/").`).
+			ArgDoc("wipe", `If true, then the host directory will be wiped clean such that it exactly matches the directory being exported, including deleting any files on the host that aren't in the exported dir. If false (the default), the contents of the directory will be merged with any existing contents of the host directory, leaving any existing files on the host that aren't in the exported directory alone.`),
```

Comment:
```suggestion
			ArgDoc("wipe", `If true, then the host directory will be wiped clean before exporting so that it exactly matches the directory being exported. If false (the default), the contents of the directory will be merged with any existing contents of the host directory, leaving any existing files on the host that aren't in the exported directory alone.`),
```

Reply:
I kind of like being extra explicit about the fact that it will delete files on the host, so leaving in that extra part but will clarify the phrasing.